app/static/js/libs/marked.js

```diff
@@ -834,7 +834,7 @@ Renderer.prototype.listitem = function(text, checked) {
   }
 
   return '<li class="task-list-item">'
-    + '<input type="checkbox" class="task-list-item-checkbox"'
+    + '<input type="checkbox" class="task-list-item-checkbox" onchange = "checkUncheck(this)"'
```

Comment:
@Frederick-S Hi, I'm not very familiar with javascript, so i kinda just worked around it. You're saying that we create a separate custom function right? I don't really know how we'd integrate it. 
It'd be great if you could help out :)